qa/run-standalone.sh

```diff
@@ -1,25 +1,87 @@
-#!/bin/sh -ex
+#!/bin/bash
 
-if [ ! -e Makefile ]; then
+if [ ! -e Makefile -o ! -d bin ]; then
     echo 'run this from the build dir'
     exit 1
 fi
 
+if [ ! -d /tmp/ceph-disk-virtualenv -o ! -d /tmp/ceph-detect-init-virtualenv ]; then
+    echo '/tmp/*-virtualenv directories not built'
+    exit 1
+fi
+
 if [ `uname` = FreeBSD ]; then
     # otherwise module prettytable will not be found
     export PYTHONPATH=/usr/local/lib/python2.7/site-packages
     exec_mode=+111
 else
+    export PYTHONPATH=/usr/lib/python2.7/dist-packages
     exec_mode=/111
 fi
 
-for f in `find ../qa/standalone -perm $exec_mode -type f`
+PATH=$(pwd)/bin:$PATH
+
+# TODO: Use getops
```

Comment:
@dzafman 
If you are going to use getopts here, then please not that FreeBSD has 2 versions.
1 in /usr/bin and 1 in /usr/local/bin.
And only the last one is going to be compatible with what youe are going to do.
So perhaps stick a define for that in the FreeBSD switch at the beginning.